cmd/crio/main.go

```diff
@@ -308,51 +308,51 @@ func main() {
 		},
 		cli.StringFlag{
 			Name:  "pause-command",
-			Usage: "name of the pause command in the pause image",
+			Usage: fmt.Sprintf("name of the pause command in the pause image (default: %q)", defConf.PauseCommand),
 		},
 		cli.StringFlag{
 			Name:  "pause-image",
-			Usage: "name of the pause image",
+			Usage: fmt.Sprintf("name of the pause image (default: %q)", defConf.PauseImage),
 		},
 		cli.StringFlag{
 			Name:  "pause-image-auth-file",
-			Usage: "path to a config file containing credentials for --pause-image",
+			Usage: fmt.Sprintf("path to a config file containing credentials for --pause-image (default: %s)", defConf.PauseImageAuthFile),
 		},
 		cli.StringFlag{
 			Name:  "signature-policy",
-			Usage: "path to signature policy file",
+			Usage: fmt.Sprintf("path to signature policy file (default: %q)", defConf.SignaturePolicyPath),
 		},
 		cli.StringFlag{
 			Name:  "root",
-			Usage: "crio root dir",
+			Usage: fmt.Sprintf("crio root dir (default: %q)", defConf.Root),
 		},
 		cli.StringFlag{
 			Name:  "runroot",
-			Usage: "crio state dir",
+			Usage: fmt.Sprintf("crio state dir (default: %q)", defConf.RunRoot),
 		},
 		cli.StringFlag{
 			Name:  "storage-driver",
-			Usage: "storage driver",
+			Usage: fmt.Sprintf("storage driver (default: %q)", defConf.Storage),
 		},
 		cli.StringSliceFlag{
 			Name:  "storage-opt",
-			Usage: "storage driver option",
+			Usage: fmt.Sprintf("storage driver option (default: %q)", defConf.StorageOptions),
 		},
 		cli.BoolFlag{
 			Name:  "file-locking",
-			Usage: "enable or disable file-based locking",
+			Usage: fmt.Sprintf("enable or disable file-based locking (default: %t)", defConf.FileLocking),
 		},
 		cli.StringSliceFlag{
 			Name:  "insecure-registry",
 			Usage: "whether to disable TLS verification for the given registry",
 		},
 		cli.StringSliceFlag{
 			Name:  "registry",
-			Usage: "registry to be prepended when pulling unqualified images, can be specified multiple times",
+			Usage: fmt.Sprintf("registry to be prepended when pulling unqualified images, can be specified multiple times (default: %q)", defConf.Registries),
```

Comment:
Isn't this going to list all of the possible registries to use from registries.conf rather than the default single registry?